Group dependencies for submodules (#67)
facebookresearch/AugLy: augly/image/README.md

# Image

## Installation   

> If you want to use the image augmentations from AugLy, you can install using the following command:   

Reviewer: ```suggestion
If you would like to use the image augmentations, please install AugLy using the following command:
```
